local_pathfinding/ompl_path.py

```diff
@@ -97,6 +100,47 @@ def update_objectives(self):
         """
         raise NotImplementedError
 
+    def choose_planner(self, planner: str, si: ob.SpaceInformation) -> ob.Planner:
```

Comment:
Make this a global function and put at the end of the file, since it does use any instance variables